packages/graphql-language-service-server/src/MessageProcessor.js

```diff
@@ -434,10 +442,8 @@ async function provideDiagnosticsMessage(
   return results;
 }
 
-function sendMessageIPC(message: any): void {
-  if (process.send !== undefined) {
-    process.send(message);
-  }
+function sendMessageIPC(message: any, writer: (message: string) => void): void {
+  writer.write(message);
 }
```

Comment:
This function doesn't really do anything any more. Should we just inline it at the call sites?

eg.

```
sendMessageIPC(response, writer);
sendMessageIPC({id: requestIDToCancel}, writer);
```

would just become:

```
writer.write(response);
writer.write({id: requestIDToCancel});
```